WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -97,3 +135,23 @@ extension SiteAddressViewController: NUXKeyboardResponder {
         keyboardWillHide(notification)
     }
 }
+
+
+// MARK: - Constants
+extension SiteAddressViewController {
+    /// Rows listed in the order they were created
+    ///
+    enum Row {
```

Comment:
This should be `private`. Also, you could just make the extension `private` and move all the methods in here instead of having to explicitly mark everything `private`.